src/styles/blocks/content.scss

```diff
@@ -0,0 +1,157 @@
+.star {
+  display: flex;
+}
+
+.stars_star {
+  display: flex;
+  margin-left: 4px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  background-position: center;
+}
+
+.stars--5 .stars_star:not(:nth-child(n + 6)),
+.stars--3 .stars_star:not(:nth-child(n + 4)),
+.stars--2 .stars_star:not(:nth-child(n + 3)),
+.stars--1 .stars_star:nth-child(1),
+.stars--4 .stars_star:not(:nth-child(n + 5)) {
+  background-image: url(../images/star-active.svg);
+}
+
+.star .stars_star:first-child {
+  margin-left: 0;
+}
+
+.card {
```

Comment:
Check the 8th point from the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md?rgh-link-date=2022-03-08T15%3A54%3A52Z)